.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Feature request
+about: Suggest an idea for this project
```

Comment:
change about to
Please suggest ideas, questions or alternate implementation suggestions here. Remote Audio Data is meant to be a collaborative effort and we hope to foster that dialogue.